mesosutil/constants.go

```diff
@@ -2,5 +2,5 @@ package mesosutil
 
 const (
 	// MesosVersion indicates the supported mesos version.
-	MesosVersion = "0.23.0"
```

Comment:
It means that we started supporting mesos 0.24.0. It says nothing about support for older versions which should be documented through git tags and the CHANGELOG.